src/docs/docker-plugin.md

```diff
@@ -59,13 +71,22 @@ Also during volume creation, customer is able to set additional options for new
 Example:
 
 ```
-docker volume create -d nexenta/nexentaedge-nfs-plugin:dev -o enableErasure=true -o erasureMode="4:2:rs" nfs01@clu1/ten1/buc1
+docker volume create -d nexenta/nexentaedge-nfs-plugin:dev -o enableErasure=true -o erasureMode="4:2:rs" ten1/buc1
 ```
 
 ### Run container with already created volume
```

Comment:
Run container and attach existing volume

Reply:
I presume that "-o enableErasure=true" is enabling Erasure Coding of the volume.

Would it be possible to name this option something that did not imply enabling expunging of the object?